src/map/skill.c

```diff
@@ -13913,21 +13913,43 @@ static int skill_unit_onplace_timer(struct skill_unit *src, struct block_list *b
 			if( battle->check_target(&src->bl,bl,BCT_ENEMY) > 0 ){
 				switch( sg->unit_id ){
 					case UNT_ZENKAI_WATER:
-						sc_start(ss, bl, SC_COLD, sg->val1*5, sg->skill_lv, skill->get_time2(sg->skill_id, sg->skill_lv));
-						sc_start(ss, bl, SC_FREEZE, sg->val1*5, sg->skill_lv, skill->get_time2(sg->skill_id, sg->skill_lv));
-						sc_start(ss, bl, SC_FROSTMISTY, sg->val1*5, sg->skill_lv, skill->get_time2(sg->skill_id, sg->skill_lv));
+						switch (rnd()%3) {
+							case 0:
```

Comment:
https://github.com/HerculesWS/Hercules/wiki/Coding-Style#chapter-1-indentation
switch and case should be on the same indent